bigtable/tests/unit/test_table.py

```diff
@@ -399,20 +402,21 @@ def test_read_row_complete(self):
         from google.cloud.bigtable.row_data import Cell
         from google.cloud.bigtable.row_data import PartialRowData
 
+        app_profile_id = 'app-profile-id'
         chunk = _ReadRowsResponseCellChunkPB(
             row_key=self.ROW_KEY,
             family_name=self.FAMILY_NAME,
             qualifier=self.QUALIFIER,
             timestamp_micros=self.TIMESTAMP_MICROS,
             value=self.VALUE,
-            commit_row=True,
+            commit_row=True
```

Comment:
Not a blocker for merge, but trailing commas are, in general, a Good Thing(TM), especially in multi-line scenarios like this one:  they make future editing simpler (e.g., if another paramter is added, one doesn't need to remember to add the comma).